root/static/scripts/tests/release-editor/fields.js

```diff
@@ -183,3 +183,28 @@ fieldTest("tracks are set correctly when the cdtoc is changed", function (t, rel
     );
     t.ok(_.last(medium.tracks()).isDataTrack());
 });
+
+fieldTest("Tracks' time are changed correctly when inputting values in the medium tracklist editing form tab", function (t, release){
```

Comment:
```suggestion
fieldTest("track times entered as integers are converted into HH:MM:SS", function (t, release){
```
might be a shorter and more accurate description of what it's testing